bundles/paikkatietoikkuna/coordinateconversion/service/ConversionService.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @class Oskari.mapframework.bundle.coordinatetool.CoordinateToolService
+ */
+Oskari.clazz.define('Oskari.coordinateconversion.ConversionService',
+/**
+ * @method create called automatically on construction
+ * @static
+ */
+function(instance) {
+    this._instance = instance;
+    this._sandbox = instance.sandbox;
+    this.urls = {};
+
+    var ajaxUrl = this._sandbox.getAjaxUrl() + '&action_route=';
+    this.urls.reverseGeocode = (ajaxUrl + 'GetConversionResult');
```

Comment:
You can replace this with Oskari.urls.getRoute('GetConversionResult'). It will return the same as sandbox.getAjaxUrl() + '&action_route=' + GetConversionResult;